server/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: "3.2"
+
+services:
+  apigateway:
+    image: rsww/apigateway:${TAG:-latest}
+    build:
+      context: .
+      dockerfile: ApiGateway/Dockerfile
+    networks:
+      - backend
+    ports:
+      - 8080:443
+    environment:
+      - ASPNETCORE_URLS=https://+:443;http://+:80
+      - ASPNETCORE_Kestrel__Certificates__Default__Password=${SECRET_PASS}
+    depends_on:
+      - identity-service
+
+  identity-service:
+    image: rsww/indentity-service:${TAG:-latest}
+    build:
+      context: .
+      dockerfile: IdentityService/Dockerfile
+    networks:
+      - backend
+      - auth_db
+    depends_on:
+      - postgres
+
+#  rabbitmq:
+#    image: "rabbitmq:3-management"
+#    environment:
+#      - RABBITMQ_DEFAULT_USER=${RABBITMQ_DEFAULT_USER}
+#      - RABBITMQ_DEFAULT_PASS=${RABBITMQ_DEFAULT_PASS}
+#      - RABBITMQ_ERLANG_COOKIE=${RABBITMQ_ERLANG_COOKIE}
+#    networks:
+#      - backend
+#    healthcheck:
+#      test: rabbitmq-diagnostics -q ping
+#      interval: 30s
+#      timeout: 30s
+#      retries: 3
+
+  postgres:
+    image: postgres:13-alpine
```

Comment:
I heard that there was an issue with Polish diacritics when the default collation of a postgres based on alpine was there. Not that I suggest a deep dive into this right now, but if it just so happens that we'd have an issue with that, maybe someone would recall.

Reply:
I think we can ignore this issue at the moment. Postgres should be used only to store info about user (login, hashed password and guid), or at the moment it's supposed do only this :) If we decide to store there some more data, maybe with Polish siacritics, then we will decide what to do with that.